<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BiographiesView</class>
 <widget class="QDialog" name="BiographiesView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>851</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Biographies</string>
  </property>
  <widget class="QListWidget" name="w_list">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>171</width>
     <height>841</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="w_name">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>281</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="w_bio">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>311</width>
     <height>751</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="w_saveBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>810</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
